exercises/two-fer/.meta/canonical-data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "exercise": "two-fer",
+  "version": "1.2.0",
+  "cases": [
+    {
+      "description": "no name given",
+      "property": "twoFer",
+      "input": {},
```

Comment:
I think what we need to do is drop the input and that should be it.

Reply:
Removed. :)